README.md

```diff
@@ -3,6 +3,8 @@
 [Orka](https://www.macstadium.com/orka) GitHub runner is a tool that integrates with GitHub to provide demand-based solution for customer workflows.
 It leverages ephemeral runners to ensure real-time execution and dynamic scaling, eliminating the need for manual provisioning and maintenance of runners.
 
+> **NOTE**: This project is not affiliated with the original [Actions Runner Controller (ARC)](https://github.com/actions/actions-runner-controller) project but uses portions of its code under the terms of the Apache License 2.0.
```

Comment:
@jasoncdavis0 - There are some differences in the README - feel free to review and suggest any changes you see fit